### ISSUE TITLE
Error in na_ontap_nfs

### ISSUE THREAD

**Issue by Gianlu** [ansible-collections/netapp.ontap#197]
### Summary

When I try to configure nfs via ansible playbook, the creation is ok while, in subseguent runs, it fails (even if the result should be "unchanged")

### Component Name

na_ontap_nfs

### Ansible Version

```console
$ ansible --version
ansible [core 2.15.3]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/vscode/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.9/dist-packages/ansible
  ansible collection location = /home/vscode/.ansible/collections:/usr/share/ansible/collections:/tmp/collections
  executable location = /usr/local/bin/ansible
  python version = 3.9.2 (default, Feb 28 2021, 17:03:44) [GCC 10.2.1 20210110] (/usr/bin/python3)
  jinja version = 3.1.2
  libyaml = True
```

### ONTAP Collection Version

```console
$ ansible-galaxy collection list
# /home/vscode/.ansible/collections/ansible_collections
Collection            Version
--------------------- -------
awx.awx               22.6.0 
community.hashi_vault 6.0.0  

# /tmp/collections/ansible_collections
Collection            Version
--------------------- -------
community.general     8.1.0  
mol.storage           1.0.0  
netapp.ontap          22.8.3
```

### ONTAP Version

```console
sridharc-vsim34::> version
NetApp Release 9.10.1P10: Sun Dec 18 16:24:34 UTC 2022
```

### Playbook

```console
- name: Configure nfs
  netapp.ontap.na_ontap_nfs:
    hostname: "{{ netapp_hostname }}"
    username: "{{ netapp_username }}"
    password: "{{ netapp_password }}"
    validate_certs: false
    use_rest: always
    state: present
    service_state: started
    vstorage_state: disabled
    vserver: "{{ netapp_svm_name | lower }}"
    nfsv3: enabled
    nfsv4: enabled
    nfsv41: enabled
```

### Steps to Reproduce

```yaml (paste below)

```

### Expected Results

I expected an unchanged or changed. nfs creation is ok. In a second run (without modifications) I have an error

### Actual Results

```console
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py:1095: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.41.0.21'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n  warnings.warn(\n/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py:1095: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.41.0.21'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n  warnings.warn(\nTraceback (most recent call last):\n  File \"/home/vscode/.ansible/tmp/ansible-tmp-1702913184.0929132-83721-9303539598101/AnsiballZ_na_ontap_nfs.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/home/vscode/.ansible/tmp/ansible-tmp-1702913184.0929132-83721-9303539598101/AnsiballZ_na_ontap_nfs.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/vscode/.ansible/tmp/ansible-tmp-1702913184.0929132-83721-9303539598101/AnsiballZ_na_ontap_nfs.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.netapp.ontap.plugins.modules.na_ontap_nfs', init_globals=dict(_module_fqn='ansible_collections.netapp.ontap.plugins.modules.na_ontap_nfs', _modlib_path=modlib_path),\n  File \"/usr/lib/python3.9/runpy.py\", line 210, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.9/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib/python3.9/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_netapp.ontap.na_ontap_nfs_payload_055qnjiu/ansible_netapp.ontap.na_ontap_nfs_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_nfs.py\", line 702, in <module>\n  File \"/tmp/ansible_netapp.ontap.na_ontap_nfs_payload_055qnjiu/ansible_netapp.ontap.na_ontap_nfs_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_nfs.py\", line 698, in main\n  File \"/tmp/ansible_netapp.ontap.na_ontap_nfs_payload_055qnjiu/ansible_netapp.ontap.na_ontap_nfs_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_nfs.py\", line 675, in apply\n  File \"/tmp/ansible_netapp.ontap.na_ontap_nfs_payload_055qnjiu/ansible_netapp.ontap.na_ontap_nfs_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_nfs.py\", line 418, in get_nfs_service\n  File \"/tmp/ansible_netapp.ontap.na_ontap_nfs_payload_055qnjiu/ansible_netapp.ontap.na_ontap_nfs_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_nfs.py\", line 558, in get_nfs_service_rest\nTypeError: argument of type 'NoneType' is not iterable\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

**Comment by csahu1** [comment]
We have DEVOPS-6556 created for this bug.

**Comment by carchi8py** [comment]
This was fixed in 22.9.0 that was released at the start of the month
